### DISCUSSION TITLE
relayed connection reconnecting

### DISCUSSION THREAD

**Discussion by PeterJanS** [immense/Remotely#381] [Q&A]
Since a day i use the remotely enviroment and i am very happy with it. Thanks for the work !

Most connection work great without a problem, but 1 machine has a connection problem:
If i start a remote session the desktop appears, but i can't do anything,  i don't see a mousepointer and it keeps 'reconnecting'. 
When i look at the status of the connection it's 'relayed.'
What can i do ?

Pj

**Comment by dhl-admin** [comment]
I have a similar problem. Some of my devices disconect and couple seconds later come online. And when i stablish the connection, said "Connection Delayed"
